src/h5web/visualizations/heatmap/MappedHeatmapVis.tsx

```diff
@@ -12,14 +12,16 @@ interface Props {
   dims: number[];
   mapperState: DimensionMapping;
   valueLabel?: string;
-  axesLabels?: (string | undefined)[];
+  axesNames?: (string | undefined)[];
+  axesLabels?: Record<string, string>;
   axesValues?: Record<string, HDF5Value>;
 }
 
 function MappedHeatmapVis(props: Props): ReactElement {
   const {
     value,
-    axesLabels = [],
+    axesNames = [],
+    axesLabels = {},
```

Comment:
Maybe we could pass `axesParams={{ some_axis_name: { label: ..., values: ...}, some_other_axis_name: {...} }}` and pick the right params object below? This would replace three props with one.